src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -9,13 +9,14 @@ import {COLORS} from '../../../utils/theme';
  * @property {String} text
  * @property {Boolean} showArrow
  * @property {Boolean} [rotateArrow]
+ * @property {Boolean} [withBorderBottom]
  */
 /**
  *
  * @param {DiscoveryTitleSeparatorProp} props
  */
 const DiscoveryTitleSeparator = (props) => {
-  const {text, showArrow, rotateArrow = false} = props;
+  const {text, showArrow, rotateArrow = false, withBorderBottom = false} = props;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `withBorderBottom` prop is well implemented. It's good to see that you've provided a default value for it, which will prevent potential issues if the prop isn't passed in some instances.
```diff
+  * @property {Boolean} [withBorderBottom]
   */
  /**
   *
   * @param {DiscoveryTitleSeparatorProp} props
   */
  const DiscoveryTitleSeparator = (props) => {
-  const {text, showArrow, rotateArrow = false} = props;
+  const {text, showArrow, rotateArrow = false, withBorderBottom = false} = props;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->